### ISSUE TITLE
Understanding increase in 1980 recruitment from last year's assessment

### ISSUE THREAD

**Issue by andrew-edwards** [pacific-hake/hake-assessment#1104]
Not really an issue, but doing here rather than clogging up Slack.
So here's two figures for recruitment scaled by 2010 recruitment:
2023 assessment:
![image](https://github.com/pacific-hake/hake-assessment/assets/6977190/50c6370d-638f-41fc-bf05-3fa67a5cdfbd)

2024 assessment:
![image](https://github.com/pacific-hake/hake-assessment/assets/6977190/c60c7ddc-93ed-4d25-994e-74b863597b71)

Look at 1980. Much higher (relatively) now than for last year.
Here are the numbers:
```
Billions of fish, median and 95% intervals:
                    1980 recruitment     2010 recruitment:
2023 assessment:    17.1 (10-34          16.9 (11-30)
2024 assessment:    19.7 (12-37)         15.8 (11-27)
```

So 1980 has gone up and 2010 has gone down a bit, hence the change in relative values (looking at medians, credible intervals in that table still mostly overlap).
I think this is caused by the penultimate bridging step, adding in the modelled temporal weight-at-age, see the red to blue step here (green then mostly covers blue), from Fig 15. Slight increase (but on log scale) for 1980, slight decrease for 2010.
![image](https://github.com/pacific-hake/hake-assessment/assets/6977190/7b51fc19-db61-48ef-b0f9-d4295a43772d)

So these slight changes in rec deviations seem to translate to changes in billions of fish (haven't done the math). Just trying to tease out the changes.

**Comment by aaronmberger-nwfsc** [comment]
Yep, agree with your assessment.

**Comment by andrew-edwards** [comment]
Great. And such changes aren't "bad", because as Kelli has pointed out, we're moving to better assumptions for weight-at-age.

**Comment by andrew-edwards** [comment]
Closing. Have included the numbers in Section 3.4.2. Presumably the extra 2 billion 1980 recruits will be further explained by the new time-varying weight-at-age and maturity values.

**Comment by andrew-edwards** [comment]
Just updating these numbers for the record, as am editing the related text:
Billions of fish, median and 95% intervals:
```
                    1980 recruitment     2010 recruitment:
2023 assessment:    17.1 (10-34)          16.9 (11-30)
2024 pre-SRG:         19.7 (12-37)         15.8 (11-27)
2024 post-SRG:       17.7 (10-35)          16.0 (11-29) 
```
So no longer have a change of 2 billion fish born in 1980.
